GameOfRunes/GameOfRunes/Utils/GameConfig.swift

```diff
@@ -22,6 +22,15 @@ enum GameConfig {
     enum Enemy {
         static let gestureBubbleOffset = CGPoint(x: 0.0, y: 90.0)
         static let removalAnimationTimePerFrame: TimeInterval = 0.075
+
+        // Note: monster difficulties must be unique
+        static let evilKnightDifficulty: Int = 1
```

Comment:
Perhaps can consider this:

var dificulty: Int {
    switch self {
    case .evilKnight:
        return 1
    case .orc1:
        return 2
    case ....
    case ...
    }
}

This ensures that every type is tagged to its associated difficulty and it can be retrieved with EnemyType.evilKnight.difficulty. It also prevents the case where .orc1 is accidentally matched with troll1Difficulty since there is no association between the type and static var.

Reply:
yup i've refactored this inside EnemyEntity. But these values must still be set in GameConfig